packages/at_lookup/lib/src/at_lookup_impl.dart

```diff
@@ -407,6 +410,11 @@ class AtLookupImpl implements AtLookUp {
     return await _process(atCommand, auth: true);
   }
 
+  Future<String> _enroll(EnrollVerbBuilder builder) async {
+    var atCommand = builder.buildCommand();
+    return await _process(atCommand, auth: true);
```

Comment:
enroll is not an authenticated verb is all cases

Reply:
change made: send command with auth = false when EnrollVerbBuilder.operation is 'request'. [cdcea33](https://github.com/atsign-foundation/at_libraries/pull/657/commits/cdcea33b7a56f6a1c260ad9e014c7577e5ee37a3)